### PR TITLE
Rename element library

### PR DESCRIPTION
**Pull request by mleivaf**
Rename plugin library: libgstinterpipes-1.0.so.1 -> libgstinterpipe.so

### CODE REVIEW

File: gst/interpipe/meson.build [hunk 1]
```diff
@@ -17,10 +17,9 @@ gst_interpipes_headers = [
 ]
 
 # Build plugin library
-gst_interpipes_lib = library('gstinterpipes-@0@'.format(apiversion),
+gst_interpipes_lib = library('gstinterpipe',
   gst_interpipes_sources,
   c_args : gst_c_args,
-  version : apiversion,
```
Comment (bill9889) [inline gst/interpipe/meson.build:23]: Why is the `apiversion` not required anymore?

**Comment by mleivaf** [comment]
> Try to make a good commit message which answers these questions
> what does it do?
> why does it do?
> where does it do it? (optional)

File: gst/interpipe/meson.build (hunk 1 above)
Comment (mleivaf) [reply gst/interpipe/meson.build:23]: The commit message was changed.

Comment (mleivaf) [reply gst/interpipe/meson.build:23]: Yes, api version is not required for GStreamer plugin library.

**Review by michaelgruner — APPROVED** [review]

**Pull request merged**